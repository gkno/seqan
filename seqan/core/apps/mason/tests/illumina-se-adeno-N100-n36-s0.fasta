>read.000000000
TTCACCACGTGACTGGCAGCGACTCATCAACAACAA
>read.000000001
CCTGTCGGCAAAGTCCACCGGAGCTCCTTCCGCGTC
>read.000000002
CGGGGGCAGATCCCATTCCTTCTCGGCCACCCAGCT
>read.000000003
GTTGAAGAGTTTGAAGTTGAGTCTCTTGGGCCGGAA
>read.000000004
TCTTTTTAGCGGGCTGCTGGCCTGTCTTGCCGATGC
>read.000000005
CCGAGTGAGCGAGCAGGATCTCCATTTTGACCGCGA
>read.000000006
ATCAACCGGTTTATTGATTAACACGTAATTACAGGG
>read.000000007
TTAGGGTATATATGGCCGAGTGAGCGAGCAGGATCT
>read.000000008
GAGTATATAAGCGCCTGTTTGAACCTGGCCGAGCGC
>read.000000009
ATTCTGGTGGAGACCACGGGGGTCAAATCCATGGTG
>read.000000010
TGTGGACAACAATGGACTTTATACTGAGCCTCGCCC
>read.000000011
TGGTGCTTCAAAATATAACCTCAATGGGCGTGAATC
>read.000000012
GAATCCTCCGGCGGAGTTTTCAGCTACCAAGTTTGC
>read.000000013
CAGGAGCGTCTGCAAGAAGATACGTCTTTTGGGGGC
>read.000000014
GGCCCAGCACCAGGGATTTGACCCCCGTGGTCTCCA
>read.000000015
TGCGGTATAACCACGCCGACGCCGAGTTTCAGGAGC
>read.000000016
CCCAGCCGAGAAAGACGGAGCCGGCGTAGGCAGGTT
>read.000000017
GACCTGGACGAGCACCTGCCGGGCAGTTCTGACTCG
>read.000000018
AGACCCACGGCTAAACAGCAAGTCCTTGTTTTGGGC
>read.000000019
TTNTGTTTAGAAACGCGCTGCTGCCGATAACAGGGT
>read.000000020
CATGCGCCACCTCGGTCACGTGATCCTGCGCCCAGC
>read.000000021
AAGACGGAGCCGGCGTAGGCAGGTTCGTAGCCGTTC
>read.000000022
TTGACCAGATCGCAGGCCGAGCAAGCAATCTCGGGA
>read.000000023
TCGCGAAGCGCAACCAAGCAGTAATGTGTAAGCTAT
>read.000000024
TCCATTTTGACCGCGAAATTTGAACGAGCAGCAGCC
>read.000000025
TGACTGCCACGGTCCCAAATCTTTCGGTGGCCACAG
>read.000000026
ACCTAGGACCCTTCAACGGACTCGACAAGGGGGAGC
>read.000000027
CCAGGCCTCGTACATCTCCTTCAACGCCGCTTCCAA
>read.000000028
AATGTTGAAATTCTGATTCATTCTCTCGCATGTCTT
>read.000000029
ATGTCCGCGGGCGGAGCGGGGCCTACCAGGTAGTCG
>read.000000030
GAATCAGAATTTCAACATTTGCTTCACGCACGGGAC
>read.000000031
CCACATGGCTGGGCGACAGAGTCATCACCACCAGCA
>read.000000032
CAGGTGGTGCTGTTCCCGTCAATCACGGCGCACATG
>read.000000033
CACCAATGGCAGACAATAACGAAGGCGCCGACGGAG
>read.000000034
CACGGCTAAACAGCAAGTCCTTGTTTTGGGCACTTC
>read.000000035
GGACCGGGGCATCACCTCCGAGAAGCAGTGGATCCA
>read.000000036
TGCGGAATCATGAACACGTCCGCCGGGAACGGAGGG
>read.000000037
AGGAAGCGTCCCAGCACCATGGATTTGACCCCCGTG
>read.000000038
GGTCCGCAGGCCCCACCGAGCGAGCGAGCGCGCAGA
>read.000000039
CCGCCGGGAACGGAGGGAGGCAGCCCTGGTGCGCAG
>read.000000040
GGACGCTGCGTCCGCCGCGTTGANGGGCTCCCCCTT
>read.000000041
TCTGGCTGTTTGGGCCGGCCACCACGGGCAAGACCA
>read.000000042
CTAATCTGACTCAGGAAGCGGCCCAGCACCATGGAT
>read.000000043
GCCCAAACAGCCAGATGGTGTTGCGCTTCCCGAACT
>read.000000044
GATTTTGCATAATTGGATGTGTACTGCACTTCGGGA
>read.000000045
TTTTGCGACATTTTGCGACACCACGTGGCCATTTAG
>read.000000046
TTTGGTCAGCGCCATGATCTTGCCGGCATTGTCCAG
>read.000000047
GACGTGTTCATGATTCCGCAATACGGCTACCTGACG
>read.000000048
AATACGGCTACCTGACGCTCAACAATGGCAGCCAAG
>read.000000049
TGGTCGTAGGCCTTGTCGTGCTCGAGGGACGCTGCG
>read.000000050
CCGGCTGATGAATCCTCTATCGACCAATACCTGTAT
>read.000000051
TTCGGCTACAGCACCCCCTGGGGGTATTTTGATTTC
>read.000000052
CGTCGATGGATCCGCGACTGAGGGGCAGGCCCGCTT
>read.000000053
TCGTCTGTAATCATGACATTGTCCAATGCAGTGTTT
>read.000000054
TGAGCTGGGTGGCCGAGAAGGAATGGGATCTGCCCC
>read.000000055
GGGGTATTTTGATTTCAACAGATTCCACTGCCACTT
>read.000000056
GAGTCAGATTAGGGACAAGCTGGTGCAGACCATTCT
>read.000000057
ACTTTATACTGAGCCTCGCCCCATTGGCACCCGTTA
>read.000000058
GACACCACGTGGCCATTTAGGGTATATATGGCCGAG
>read.000000059
GCAGTGGAATCTGTTGAAATCAAAATACCCCCAGGG
>read.000000060
CTGGAGCATGACTTTGGCAAGGTGACAAAGCAGGAA
>read.000000061
GAACCTCTCGGTCTGGTTGAGGAAGGCGCTAAGACG
>read.000000062
TCGTGACCTCCTTGACTTGGATGTTGAAGAGTTTGA
>read.000000063
GGACCGCGAGTTGGAAGCGGCGTTGAAGGAGATGTA
>read.000000064
CGTGCTGGTAAGGTTATTAGCGATGGTTGTGACGCC
>read.000000065
AACCTCTCTGAGGGCATTCGCGAGTGGTGGGACTTG
>read.000000066
AAGCATGCCCGCGTGACGAGAACATTTGTTTTGGTA
>read.000000067
AGCCCAAAAACTGGCTACCTGGACCCTGTTATCGGC
>read.000000068
GCTGCAGCTTCTCGGCCACGGTCAGGGGTGCCTGCT
>read.000000069
GGCAACTCCATCACTAGGGGTAACCCGATGACGTAA
>read.000000070
TCTGACATGGATCTGAATCTGATTGAACAGGCACCC
>read.000000071
CCAACACCAACATGTGCGCCGTGATTGACGGGATCA
>read.000000072
CGGCTGATGAATCCTCTCATCGACCAATACCTATAT
>read.000000073
GCGCACATGTTGGTGTTGGAGGTGACGATCACGGGG
>read.000000074
GGCCGGCAGAGCAGAGCTCTGCCGTCTGCGGACCTT
>read.000000075
CCCGCACCTGGGCCTTGCCCACCTACAATAACCACC
>read.000000076
TACGGCTGCGTCAACTGGACCAATGAGAACTTTCCC
>read.000000077
CAAGACCAACATCGCGGAAGCCATCGCCCACGTGCC
>read.000000078
ATCCAATTATGCAAAATCTGCCAACGTTGATTTTAC
>read.000000079
AAGACCCGCCCCCGATGACGCGGATAAAAGCGAGCC
>read.000000080
GGTGACAAAGCAGGAAGTCAAAGAGTTCTTCCGCTG
>read.000000081
TCAAGGAGGTCACGACGAATGATGGCGTCACAACCA
>read.000000082
CACCAATGGCAGACAATAACGAAGGCGCCGACGGAG
>read.000000083
CTTCCAACTCGCGGTCCCAGATCAAGGCCGCTCTGG
>read.000000084
TGCGCGCTCGCTCGCTCGGTGGGGCCTGCGGACCAA
>read.000000085
CCTACCAGGTAGTCGGGCGCGGATTTGGTCACCGCC
>read.000000086
AGCAGAGCTCTGCCGTCTGCGGACCTTTGGTCCGCA
>read.000000087
TCAACGTGGACCTGGATGACTGTGTTTCTGAGCAAT
>read.000000088
TGTCCTCGAGCCAATCTGGAAGATAACCATCGGCAG
>read.000000089
GGGTCTTTTGTTGGCTCCACCCTTTCTGACGTAGAA
>read.000000090
TAGCCGTATTGCGGAATCATGAACACGTCCGCCGGG
>read.000000091
CACGTGACTGGCAGCGACTCATCAACAACAATTGGG
>read.000000092
CTTCTTATCTTATCGGTTACCATGGTTATAGCTTAC
>read.000000093
ACCCACGTCAGCCAGACCCAGGAGCAGAACAAGGAG
>read.000000094
CATTTAGGGTATATATGGCCGAGTGAGCGAGCAGGA
>read.000000095
TCAGGAAGCGGCCCAGCACCATGGATTTGACCCCCG
>read.000000096
GCGTCCCTCGAGCACGACAAGGCCTACGACCAGCAG
>read.000000097
TCACCGCGAACCAGTTGGGCAGGGTCGGCTCGATCC
>read.000000098
GCCGGGAACGGAGGGAGGCAGCCCTGGTGCGCAGAG
>read.000000099
GTCTCCTGTCCTTCTTATCTTATCGGTTACCATGGT
