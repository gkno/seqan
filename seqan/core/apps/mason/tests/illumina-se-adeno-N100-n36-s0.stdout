  Building haplotype...
    contig # 1/1
  Finished haplotype creation in 7.1306e-05s
